spyder/plugins/remoteclient/widgets/connectionstatus.py

```diff
@@ -306,9 +307,43 @@ def _set_text_in_labels(self, status):
         )
 
     def _set_icon(self, status):
-        self._image_label.setPixmap(
-            self.svg_to_scaled_pixmap(STATUS_TO_ICON[status], rescale=1)
-        )
+        # Use the icon manager to get properly colorized icons
+        icon = ima.get_icon(STATUS_TO_ICON[status])
```

Comment:
This new code needs to be used in place of the current `svg_to_scaled_pixmap` so that other icons can make use of it (like the Spyder dialog in the About dialog).